frontends/main/src/app-pages/OnboardingPage/OnboardingPage.test.tsx

```diff
@@ -87,166 +90,66 @@ const queryBackButton = () => screen.queryByRole("button", { name: "Back" })
 const queryFinishButton = () => screen.queryByRole("button", { name: "Finish" })
 
 describe("OnboardingPage", () => {
```

Comment:
This used to be organized as
```ts
describe("OnboardingPage", () => {
  describe("Page 1", () => {
    /* tests */
  })
  describe("Page 2", () => {
    /* tests */
  })
  // etc
})
```

The tests were really only checking navigation.

I moved it into `each` blocks so that if we add/remove pages it's easier to update.